pip-requirements.txt

```diff
@@ -1,5 +1,4 @@
 apache-beam[gcp]
-zarr
```

Comment:
This is a duplicate entry in the pip-requirements file, the other entry has a minimum version. With this line here, you (or at least I) can't `pip install -r` this file.